ui/src/main/webapp/home/index.js

```diff
@@ -129,7 +130,7 @@ const TileLink = ({ to, title, subtitle, children }) => (
             style={{width: '100%', height: '100%'}}>
 
             <p className={styles.titleTitle}>{title}</p>
-            {children}
+            {React.createElement(Icon, { style: { color: muiTheme.palette.primary1Color, width: '50%', height: '50%' } })}
```

Comment:
What is happening here? Is the Icon being wrapped by a React component that you style?